tensorpack/tfutils/symbolic_functions.py

```diff
@@ -272,3 +272,18 @@ def shapeless_placeholder(x, axis, name):
         shp[a] = None
     x = tf.placeholder_with_default(x, shape=shp, name=name)
     return x
+
+def softmax_cross_entropy_with_ignore_label(logits, label, class_num):
```

Comment:
Same for symbolic functions. We'll not add more symbolic functions and most of the existing ones are deprecated with a "Please implement it by yourself" notice. These functions are specific to applications and should come with the applications.
If a symbolic function is not common and general enough to be included by tensorflow itself, there is no reason why tensorpack should include it.